package.json

```diff
@@ -10,14 +10,19 @@
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
+    "acorn": "^1.1.0",
+    "es6-promises": "^1.0.10",
```

Comment:
We already added `es6-promise` (not the s lacking) in https://github.com/mozilla/buddyup/commit/029f6014cd3eadb0043e175d8769522008c7d1f8#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R13. Could we update this pull request to use that nom module instead?